ChangeLog.md

```diff
@@ -1,3 +1,12 @@
+# Changes in 1.1.7
+
+* Add a [factory](incl/../include/clap/factory) folder for better organization.
+
+## Draft extensions
+
+* [preset-discovery.h](include/clap/preset-discovery.h): new extension which allows the host to index the plugin's preset which are stored on disk.
+* [context-menu.h](include/clap/ext/dra/../draft/context-menu.h): add "title" menu entry
```

Comment:
```suggestion
* [context-menu.h](include/clap/ext/dra/../draft/context-menu.h): add "title" menu entry

## Draft factories

* [preset-discovery.h](include/clap/factory/draft/preset-discovery.h): new factory which allows the host to index the plugin presets which are stored on disk.
```

Reply:
A bit off-topic: How long are old changelog hyperlinks supposed to work? Or should they rather be maintained or removed with newer versions or whenever mentioned files are moved/renamed?
In general the '..' notation might reduce the amount of maintenance work or at least extend the lifetime of the hyperlinks.

Reply:
They should work with the version at which they were released.
We won't update the past changelog entries after a release.